src/translations/en.json

```diff
@@ -552,6 +552,7 @@
   "PayoutDetailsForm.lastNameLabel": "Last name",
   "PayoutDetailsForm.lastNamePlaceholder": "Doe",
   "PayoutDetailsForm.lastNameRequired": "This field is required",
+  "PayoutDetailsForm.missingStripeKey": "You need to configure the Stripe publishable key to continue.",
```

Comment:
Providers see this message, so I think this should be changed to something like:
"Marketplace operator has not configured Stripe publishable key to this marketplace. Unfortunately, you can't publish your listing yet."